### PR TITLE
Remove hard coded user

### PR DESCRIPTION
**Pull request by lcarva**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -44,6 +41,20 @@ function createWindow() {
...
+function checkUser() {
+	const manager = require("./node_modules/thelounge/src/clientManager");
+
+	const users = manager.prototype.getUsers();
+
+	if (!users.length) {
+		electron.dialog.showMessageBox({
+			type: "warning",
+			buttons: ["OK"],
+			message: "There are currently no users. Create one with lounge add <name>."
```
Comment (MaxLeiter) [inline index.js]: Could we do this via the UI?

Comment (MaxLeiter) [reply index.js]: I.e. lounge add user

Comment (lcarva) [reply index.js]: Message copied directly from what `lounge start` outputs in this scenario. 

What do you mean via UI?

**Comment by MaxLeiter** [comment]
Text boxes to make a new user

**Comment by lcarva** [comment]
> Text boxes to make a new user

With that comes form validation and everything that goes along with it. If the long term goal is to generate a local user on the fly, it's probably not worth chasing this.

**Comment by xPaw** [comment]
Yeah this is slightly subpar solution to ask users to run a CLI command.

I'm not sure how we would solve it correctly though. If we ask users if they want to host a local server or connect to a remote one, then entering credentials on remote server is not a problem (if we will want to store them in OS credential store however, we will need a way of passing in user/password into the DOM  somehow)

**Comment by lcarva** [comment]
I'm no longer actively working on this.

**Pull request closed without merging**